src/components/pages/CourseSearch.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import Requestor from '../Requestor';
+
+const CourseSearch = () => {
+  const [courses, setCourses] = useState([]);
+
+  // Get courses
+  useEffect(() => {
+    getCourses();
+    // eslint-disable-next-line
+  });
+
+  const getCourses = async () => {
+    const coursesResponse = await Requestor.get(
+      '/courses',
+      {},
+      {
+        filters: {},
+        offset: 0,
+        size: 5
+      }
+    );
+    setCourses(coursesResponse.data);
+  };
+
+  const showCourses = courses.map(course => (
+    <tr key={course.id}>
+      <th>{course.id}</th>
+      <th>{course.study_abroad_institution}</th>
+      <th>{course.program_name}</th>
+      <th>{course.year}</th>
+      <th>{course.kzoo_course_name}</th>
+      <th>{course.kzoo_discipline}</th>
+      <th>{course.host_insti_course_number}</th>
+      <th>{course.host_insti_course_name}</th>
+    </tr>
+  ));
+
+  return (
+    <Fragment>
+      <table style={{ width: '100%' }}>
+        <tbody>
+          <tr>
+            <th>ID</th>
```

Comment:
Let's put these in a constant file. So under `src/components/pages/course/course.constant.js`, make

```js
export const coursesColumnConfig = [
  {
    id: 'id',
    display: 'ID',
    // we will add more later as we work on `Table.js` component
  },
  // ...
];
```